Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -318,7 +318,7 @@ private void InitializeServiceClient(TelemetryConfiguration configuration)
         private static string GetInstanceName(TelemetryConfiguration configuration)
         {
             // we need to initialize an item to get instance information
-            var fakeItem = new MetricTelemetry();
```

Comment:
Can you please check with chucknorris if this is exactly the same for all cases? That there are no checks in initializers that won't add the context we need to an Event?